go.mod

```diff
@@ -23,8 +23,15 @@ require (
 )
 
 require (
+	github.com/cloudevents/sdk-go/protocol/nats/v2 v2.10.1
+	github.com/sirupsen/logrus v1.8.1
+)
```

Comment:
this looks weird... should these not be part of the require above?
Maybe run `go mod tidy`

Reply:
`go mod tidy` did not work, but I moved the dependencies up manually followed by `go mod tidy`, which seems to work.